src/pages/Actions/ActionEvent/EventInfo/FormAddPictures.tsx

```diff
@@ -94,7 +94,7 @@ const FormAddPictures = ({ eventId, updateGallery, picturesCount }: Props) => {
             onFinish={handleSubmit}
         >
 
-            <Form.Item wrapperCol={{ span: 12, offset: 6 }}
+            <Form.Item wrapperCol={{  }}
```

Comment:
Why didn't you just delete the wrapperCol property?

Reply:
its doesn't work without it